services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
+        movie_id: int,
+        cinema_hall_id: int
+) -> QuerySet:
+
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id,
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> QuerySet:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: str = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> QuerySet:
+    session = MovieSession.objects.get(id=session_id)
+
+    if show_time:
+        session.show_time = show_time
+
+    if movie_id:
+        session.movie_id = movie_id
+
+    if cinema_hall_id:
+        session.cinema_hall_id = cinema_hall_id
+
+    session.save()
+    return session
+
+
+def delete_movie_session_by_id(session_id: int) -> QuerySet:
```

Comment:
The function `delete_movie_session_by_id` should not return `QuerySet`. Consider returning `None` or a confirmation message instead.